.gitignore

```diff
@@ -48,10 +48,6 @@ archive
 todo
 scratchpad
 
-# Ignore GPT Engineer files
```

Comment:
why are we adding gpte-creataed projects to the repo? :)

Reply:
It's an example project to run improve against

Reply:
I think this folder should be a list of useful examples which people can quickly run commands against when trying got engineer out or manually testing

Reply:
Think it makes sense. We could encourage people to do `cp projects/example their-own-folder` Similar to a `.env.example` flow.